protocol/README.md

```diff
@@ -207,14 +190,23 @@ Our current implementation contains a light fork of CosmosSDK. The fork can be f
 
 Daemons are background processes that run in go-routines to do asyncronous work. Daemons can be configured with their respective flags, e.g. `price-daemon-enabled` or `price-daemon-delay-loop-ms`.
 
+TODO(CORE-512): update daemon flags
+
+### Bridge Daemon
+
+TODO(CORE-512): add details
+
+### Liquidation Daemon
+
+TODO(CORE-512): add details
+
 ### Pricefeed Daemon
 
 The Pricefeed Daemon is responsible for ingesting prices from 3rd party exchanges like Binance and sending these prices to the application where they are then used by the Prices module. The Pricefeed daemon is started by default when the application starts.
 
-The path to the configuration file used for querying exchanges is dictated by the flag `pricefeed-exchange-config-file`. The default configuration file is located within `v4` at the path `daemons/configs/default_pricefeed_exchange_config.go`. The file utilizes a template that is populated from `daemons/pricefeed/client/constants/static_exchange_startup_config.go`. The first exchange within the default configuration file additionally has inline comments explaining what each field is for.
```

Comment:
we got rid of this as part of [this PR](https://github.com/dydxprotocol/v4-chain/pull/627/files#diff-e38b2465703f6bc6c6fbcd86c689425295c30421c5835d2b95227c0311905b3b)